tests/test_schema.py

```diff
@@ -578,87 +137,27 @@ def test_add_schema_entry():
             assert False
 
 
-@pytest.mark.skip("requires AsnModel")
-def test_table_size_zero():
-    with AsnModel() as dm:
-        assert len(dm.asn_table) == 0
-
-
-@pytest.mark.skip("requires MultiSlitModel")
-def test_copy_multslit():
-    model1 = MultiSlitModel()
-    model2 = MultiSlitModel()
-
-    model1.slits.append(ImageModel(np.ones((1024, 1024))))
-    model2.slits.append(ImageModel(np.ones((1024, 1024)) * 2))
-
-    # Create the ouput model as a copy of the first input
-    output = model1.copy()
-
-    assert len(model1.slits) == 1
-    assert len(model2.slits) == 1
-    assert len(output.slits) == 1
-
-    assert model1.slits[0].data[330, 330] == 1
-    assert output.slits[0].data[330, 330] == 1
-    assert id(model1.slits[0].data) != id(output.slits[0].data)
-
-    output.slits[0].data = model1.slits[0].data - model2.slits[0].data
-
-    assert model1.slits[0].data[330, 330] == 1
-    assert output.slits[0].data[330, 330] == -1
-
-
-@pytest.mark.skip("requires MultiSlitModel")
-def test_multislit_move_from_fits():
-    from astropy.io import fits
-
-    hdulist = fits.HDUList()
-    hdulist.append(fits.PrimaryHDU())
-    for i in range(5):
-        hdu = fits.ImageHDU(data=np.zeros((64, 64)), name='SCI')
-        hdu.ver = i + 1
-        hdulist.append(hdu)
-
-    hdulist.writeto(TMP_FITS, overwrite=True)
-
-    n = MultiSlitModel()
-    with MultiSlitModel(TMP_FITS) as m:
-        n.slits.append(m.slits[2])
-
-        assert len(n.slits) == 1
-
-
-@pytest.mark.skip("requires CollimatorModel")
 def test_validate_transform():
     """
     Tests that custom types, like transform, can be validated.
     """
-    m = CollimatorModel(model=models.Shift(1) & models.Shift(2),
-                        strict_validation=True)
-    m.meta.description = "Test validate a WCS reference file."
-    m.meta.author = "ND"
-    m.meta.pedigree = "GROUND"
-    m.meta.useafter = "2018/06/18"
-    m.meta.reftype = "collimator"
+    m = TransformModel(
+        transform=models.Shift(1) & models.Shift(2),
+        strict_validation=True
+    )
```

Comment:
Maybe test some of the metadata here too?  I think that was the purpose of the other test, as it also tested some of the required metadata for the reference file?

Reply:
I removed most of the metadata validation tests from this file because there's good coverage of the same code in test_validation.py.  Probably that module and this should be merged at some point?